config/vufind/AdminMenu.yaml

```diff
@@ -0,0 +1,47 @@
+# If this file is empty or missing, default settings will be used, matching
+# the examples provided below.
+#
+# Please note that the current implementation of AdminMenu only supports a subset of
+# the options supported by AccountMenu.
```

Comment:
It probably would be a good thing if we could eventually refactor menu rendering to be based on the same templates as much as possible. And that would mean the options in all menu configuration files would be the same. But as this would be a big change I thought it might be good to go ahead in smaller steps.